TFlippy_TerritoryControl_Items_Dev/Entities/Items/Drugs/Tea/Tea_effect.as

```diff
@@ -0,0 +1,83 @@
+//Script by Gingerbeard
+#include "HittersTC.as";
+
+const string[] scriptnames = 
```

Comment:
IMO this isnt the best way of doing this

Better way would be to have all drugs implement some sort of onDie/Removal command so all you need to do is call this.SendCommand(x) to have all drugs removed on their own. This would make adding new drugs easier in the future.